base/pcre.jl

```diff
@@ -8,18 +8,17 @@ import ..RefValue
 
 # include($BUILDROOT/base/pcre_h.jl)
 include(string(length(Core.ARGS) >= 2 ? Core.ARGS[2] : "", "pcre_h.jl"))
-
-const PCRE_LIB = "libpcre2-8"
+using .Base.JLLAdapters: get_libpcre2_8
```

Comment:
How about renaming `get_libpcre2_8` to just `get_libpcre2`? I realize you probably used `get_libpcre2_8` because the soname is `libpcre2-8`, but won't that just cause code churn here if this ever increases? The old name was `PCRE_LIB` after all, i.e. it also didn't bother to use the full soname...

Reply:
`libpcre2_8` is the canonical name because it's the 8-bit version of `libpcre2`, as opposed to `libpcre2_16` and `libpcre2_32`.  ;)

Reply:
ahhhh OK :-). Then I withdraw my suggestion which clearly was nonsense. Sorry, and thanks for the explanation!